flexeval/core/chat_dataset/chatbot_bench.py

```diff
@@ -39,6 +39,7 @@ def __init__(
         path_or_name: str,
         ref_path_or_name: str | None = None,
         need_ref_categories: list[str] | None = None,
+        load_only_first_n: int | None = None,
```

Comment:
duplicate of #138 ?

Reply:
Thank you for the catch!
I rebased the PR.